bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java

```diff
@@ -481,7 +506,7 @@ public T getResult() {
             return result;
         }
 
-        public PlacementPolicyAdherence isAdheringToPolicy() {
+        public PlacementPolicyAdherence getAdheringToPolicy() {
```

Comment:
Why change this? If you need, you can add a new method and keep the original method.

Reply:
thanks.